protocols/request-response/tests/ping.rs

```diff
@@ -49,11 +49,11 @@ fn ping_protocol() {
     let cfg = RequestResponseConfig::default();
 
     let (peer1_id, trans) = mk_transport();
-    let ping_proto1 = RequestResponse::new(PingCodec(), protocols.clone(), cfg.clone());
+    let ping_proto1 = RequestResponse::new(PingCodec(), protocols.clone(), cfg.clone()).throttled();
```

Comment:
I think it would be good to have a separate, dedicated test for the throttling.

Reply:
Done in 52d03bd82f29262f76cb0891474722fa90983dbe.